src/containers/Sidebar/index.tsx

```diff
@@ -14,6 +13,7 @@ import { getActiveAccount } from '@selectors'
 import { RootState } from '@store'
 import { PersistedAccount, RootRoutes } from '@types'
 import { push } from 'connected-react-router'
+import { connect } from 'react-redux'
```

Comment:
are you able to turn off your editor/workspace auto-formatting? it's often making changes in files that aren't related, such as this, which is kind of unnecessary

Reply:
This one was a bit weird. I think it was a merge conflict which got funky